### PR TITLE
Update Routes Module to latest MOST

### PR DESCRIPTION
**Pull request by cmfcmf**
This is a PR onto the `routing-overhaul` branch, which is going to update the MOST module to the latest generator version.

### CODE REVIEW

File: src/system/RoutesModule/Resources/config/services.yml [hunk 1]
```diff
@@ -1,144 +1,110 @@
...
-    # Util classes
-    zikularoutesmodule.model_helper.class: Zikula\RoutesModule\Util\ModelUtil
-    zikularoutesmodule.controller_helper.class: Zikula\RoutesModule\Util\ControllerUtil
-    zikularoutesmodule.view_helper.class: Zikula\RoutesModule\Util\ViewUtil
-    zikularoutesmodule.workflow_helper.class: Zikula\RoutesModule\Util\WorkflowUtil
-    zikularoutesmodule.listentries_helper.class: Zikula\RoutesModule\Util\ListEntriesUtil
-
-    
-    # Log processor
-    zikularoutesmodule.log.processor.class: Monolog\Processor\PsrLogMessageProcessor
```
Comment (cmfcmf) [inline src/system/RoutesModule/Resources/config/services.yml]: @guite Are these removed on purpose? Travis fails with
>  The service "zikularoutesmodule.installer_listener" has a dependency on a non-existent parameter "zikularoutesmodule.installer_listener.class". Did you mean one of these: "zikulasettingsmodule.module_listener.class", "zikulausersmodule.user_listener.class", "zikularoutesmodule.console_command_listener.class"?

Comment (cmfcmf) [inline src/system/RoutesModule/Resources/config/services.yml]: Nevermind, the issue occurs due to the custom definitions in the `Resources/config/custom_services.xml` file.

File: src/system/RoutesModule/Api/Base/AdminApi.php [hunk 2]
```diff
@@ -14,12 +14,12 @@
 
 use ModUtil;
 use SecurityUtil;
-use Zikula_AbstractApi;
+use Zikula\Core\Api\AbstractApi;
```
Comment (craigh) [inline src/system/RoutesModule/Api/Base/AdminApi.php]: you just voted to eliminate this class....

Comment (cmfcmf) [inline src/system/RoutesModule/Api/Base/AdminApi.php]: Yes, but the Most generator still uses it. This is why its marked WIP.

Craig Heydenburg <notifications@github.com> schrieb am Sa., 5. Sep. 2015
20:06:

Comment (Guite) [inline src/system/RoutesModule/Api/Base/AdminApi.php]: https://github.com/Guite/MostGenerator/commit/c2447deffb6ac2cd561c6cb3595b083f32ccec10

**Pull request merged**